graf2d/asimage/src/libAfterImage/ungif.c

```diff
@@ -151,7 +167,11 @@ get_gif_image_desc( GifFileType *gif, SavedImage *im )
 		memcpy( &(im->ImageDesc), &(gif->Image), sizeof(GifImageDesc));
 		if( gif->Image.ColorMap )
 		{
+#if (GIFLIB_MAJOR>=5)
+			im->ImageDesc.ColorMap = GifMakeMapObject(gif->Image.ColorMap->ColorCount, NULL);
+#else
 			im->ImageDesc.ColorMap = MakeMapObject(gif->Image.ColorMap->ColorCount, NULL);
+#endif
 			fseek( gif->UserData, start_pos+9, SEEK_SET ); 
 			if(fread( im->ImageDesc.ColorMap->Colors, 1, gif->Image.ColorMap->ColorCount*3, gif->UserData)){;};
```

Comment:
This is a change introduced by the ROOT team, so I left it like that. It's probably to quiet down compiler warnings about unused return value of `fread()`.